packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/JavaScriptChannel.java

```diff
@@ -18,25 +20,39 @@
 class JavaScriptChannel {
   private final MethodChannel methodChannel;
   private final String javaScriptChannelName;
+  private final Handler platformThreadHandler;
 
   /**
    * @param methodChannel the Flutter WebView method channel to which JS messages are sent
    * @param javaScriptChannelName the name of the JavaScript channel, this is sent over the method
    *     channel with each message to let the Dart code know which JavaScript channel the message
-   *     was sent through
+   * @param platformThreadHandler
```

Comment:
The old java doc was deleted unintentionally i think

Reply:
thanks for the catch